app/src/main/java/de/danoeh/antennapod/ui/swipeactions/SwipeActionsDialog.java

```diff
@@ -98,7 +101,8 @@ public void show(Callback prefsChanged) {
             case PlaybackHistoryFragment.TAG:
                 forFragment = context.getString(R.string.playback_history_label);
                 break;
-            default: break;
+            default:
+                break;
```

Comment:
There are a bunch of unrelated changes in this file. Please keep your changes as focused as possible, without touching unrelated lines.